pokerpoll/pokerpoll.py

```diff
@@ -36,6 +36,7 @@ def __init__(self, bot):
                     self.bot.say("wot fukin time is that diked?")
                     return
 
+<<<<<<< HEAD
```

Comment:
this is the git commit conflict format, it shows the two states you're merging - HEAD and 8c5d9c (???)
you're supposed to manually resolve it by choosing the parts from each that you want, and removing the markers (<<<< ==== >>>>)